lib/Doctrine/deploy/AddServiceGroups.php

```diff
@@ -36,9 +45,10 @@
                 $scope = (string) $value;
                 break;
             default:
```

Comment:
Do we need a `default` if it's just doing a `break`? (N.B. I'm not good at PHP, so that's a genuine question!)

Reply:
what key was hitting the `default` case and causing the LogicException to be thrown? Just wondering if that should in fact be handled.

Reply:
i believe GOCDB_PORTAL_URL if i remember rightly

Reply:
So from speaking to @ineilson who added `lib/Doctrine/deploy/AddServiceGroups.php`, it was originally added to handle a stripped down subset of the GOCDB PI XML for the get_service_group method that's contained within the simpleSampleData. 

Now that the sample data is being refreshed (#394), the simple sample data can be deleted and this XML handling should be extended to handle the full XML output.

But those can be handled in future issues.

Reply:
ok, so what should i do next with this pull request?

Reply:
ok!

Reply:
> once [this](https://github.com/GOCDB/gocdb/pull/393/files#diff-267c881b565179a711a7be52ccfb482aae95438e1cf8eb805b6a0a803306f8ed) comment has been added, I will merge the PR and open the follow up issues 😄

just leaving this conversation unresolved to remind me of this 😋

Reply:
> lib/Doctrine/deploy/AddServiceGroups.php

See #415 and #416